### PR TITLE
CC Mass purchase

### PR DESCRIPTION
**Pull request by Vaedz7**
Added Mass Purchase

Features
- Buy out shop
- Purchase specific number of items
- Option to use LP
- Safely skip Full Bag

### CODE REVIEW

File: GeneratorConfig/Categories.json [hunk 1]
```diff
@@ -13,6 +13,11 @@
         "Name": "PokemonBDSP",
         "Display": "BDSP",
         "Settings": [ "PokemonBDSP_Settings" ]
+    },
```
Comment (pifopi) [inline GeneratorConfig/Categories.json:16]: I think you did not remove the files you wanted to remove (as this Pull Request seems to contains all your changes)

Comment (Vaedz7) [reply GeneratorConfig/Categories.json:16]: Shoot, I’ll fix that

Comment (Vaedz7) [reply GeneratorConfig/Categories.json]: I've removed all of the SV Native Programs

**Comment by Mysticial** [comment]
Can you rebase this and revert all the changes to the MC stuff since this is for CC?

**Comment by Vaedz7** [comment]
> Can you rebase this and revert all the changes to the MC stuff since this is for CC?

Sure will do

**Pull request closed without merging**